### DISCUSSION TITLE
Problem Getting Up and Running

### DISCUSSION THREAD

**Discussion by SG876** [rdkit/rdkit#4276] [Q&A]
I was able to install RDKit using:
   conda install -c conda-forge rdkit
I got a bunch of messages saying the install was successful.  I can see RDKit using:
   conda list
But RDKit instructions aren't accepted.  If issue:
   m = Chem.MolFromSmiles('CC(=O)NCCC1=CNc2c1cc(OC)cc2')
I get an error saying that "chem" is not defined.  Thank you for any suggestions.

**Comment by vfscalfani** [comment]
Hi @SG876. Did you import the RDKit Chem module first? Try this and see if it works:

```
from rdkit import Chem
m = Chem.MolFromSmiles('CC(=O)NCCC1=CNc2c1cc(OC)cc2')
m
```
See also: https://www.rdkit.org/docs/GettingStartedInPython.html

Vin

**Reply by SG876** [reply]
That's it.  Is there a general command that I can issue so that it will import all of RDKit (not just Chem)?

**Reply by greglandrum** [reply]
There is not. The RDKit provides a very broad set of functionality and there's no way to get it all at once.
Having said that, you can get a large fraction of the chemistry functionality with the import:
`from rdkit.Chem import AllChem`
